src/ResourceManager/ResourceGraph/Commands.ResourceGraph/AzureRM.ResourceGraph.psd1

```diff
@@ -0,0 +1,132 @@
+#
+# Module manifest for module 'AzureRM.ResourceGraph'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 9/6/2018
+#
+
+@{
+
+        # Script module or binary module file associated with this manifest.
+        # RootModule = ''
+
+        # Version number of this module.
+        ModuleVersion = '0.1.0'
+
+        # Supported PSEditions
+        # CompatiblePSEditions = @()
+
+        # ID used to uniquely identify this module
+        GUID = 'a290eead-f293-4588-b61e-b465b237591e'
+
+        # Author of this module
+        Author = 'Microsoft Corporation'
+
+        # Company or vendor of this module
+        CompanyName = 'Microsoft Corporation'
+
+        # Copyright statement for this module
+        Copyright = 'Microsoft Corporation. All rights reserved.'
+
+        # Description of the functionality provided by this module
+        Description = 'Microsoft Azure PowerShell - Azure Resource Graph cmdlets. Allows querying resources with a powerful query language.'
+
+        # Minimum version of the Windows PowerShell engine required by this module
+        PowerShellVersion = '5.0'
+
+        # Name of the Windows PowerShell host required by this module
+        # PowerShellHostName = ''
+
+        # Minimum version of the Windows PowerShell host required by this module
+        # PowerShellHostVersion = ''
+
+        # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+        DotNetFrameworkVersion = '4.5.2'
+
+        # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+        # CLRVersion = ''
+
+        # Processor architecture (None, X86, Amd64) required by this module
+        # ProcessorArchitecture = ''
+
+        # Modules that must be imported into the global environment prior to importing this module
+        RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.5.1'; })
```

Comment:
@demyanenko we just released a new version of Profile, so you'll need to update this to 5.6.0